containers/daphne.Dockerfile

```diff
@@ -64,7 +64,7 @@ LABEL "org.opencontainers.image.version"="$TIMESTAMP"
 LABEL "org.opencontainers.image.created"="${CREATION_DATE}"
 LABEL "org.opencontainers.image.revision"="${GIT_HASH}"
 RUN apt-get -qq -y update && apt-get -y upgrade && apt-get -y --no-install-recommends install  \
-    libtinfo6 libssl1.1 zlib1g python3-numpy python3-pandas \
+    libtinfo6 libssl1.1 zlib1g python3-numpy python3-pandas lit\
```

Comment:
lit is not a Ubuntu deb package - please remove this. Intended functionality is already provided by the updated Docker images (on the main branch) to fix the tests of this PR

Reply:
This is indeed a mistake, I'll remove this.